godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Basis.kt

```diff
@@ -159,15 +163,17 @@ class Basis(var x: Vector3, var y: Vector3, var z: Vector3) : CoreType {
         return getEulerYxz()
     }
 
-    // getEulerXyz returns a vector containing the Euler angles in the format
-    // (a1,a2,a3), where a3 is the angle of the first rotation, and a1 is the last
-    // (following the convention they are commonly defined in the literature).
-    //
-    // The current implementation uses XYZ convention (Z is the first rotation),
-    // so euler.z is the angle of the (first) rotation around Z axis and so on,
-    //
-    // And thus, assuming the matrix is a rotation matrix, this function returns
-    // the angles in the decomposition R = X(a1).Y(a2).Z(a3) where Z(a) rotates
+    /**
+     * getEulerXyz returns a vector containing the Euler angles in the format
+     * (a1,a2,a3), where a3 is the angle of the first rotation, and a1 is the last
+     * (following the convention they are commonly defined in the literature).
+     *
+     * The current implementation uses XYZ convention (Z is the first rotation),
+     * so euler.z is the angle of the (first) rotation around Z axis and so on,
+     *
+     * And thus, assuming the matrix is a rotation matrix, this function returns
+     * the angles in the decomposition R = X(a1).Y(a2).Z(a3) where Z(a) rotates
+     */
```

Comment:
Maybe specify the return type with `@returns ...` same for all others i mentioned before